src/com/dabomstew/pkrandom/romhandlers/AbstractRomHandler.java

```diff
@@ -4658,8 +4737,11 @@ private boolean isCyclic(Pokemon pk, Set<Pokemon> visited, Set<Pokemon> recStack
      *            Method to run on all base or no-copy Pokemon
      * @param epAction
      *            Method to run on all evolved Pokemon with a linear chain of
+     * @param ignoreNoCopyFlag
+     *            If true, treat no-copy pokemon like other evolutions
      */
-    private void copyUpEvolutionsHelper(BasePokemonAction bpAction, EvolvedPokemonAction epAction) {
+    private void copyUpEvolutionsHelper(BasePokemonAction bpAction, EvolvedPokemonAction epAction,
+                                        boolean ignoreNoCopyFlag) {

```

Comment:
Just in the interest of making this a bit less confusing, maybe this new input parameter also should be named `dontCopySplitEvos` and not be inverted when it's used?